feat: migrate from pop-up-api to popover-api (#4252)
GoogleChrome/developer.chrome.com: site/en/blog/pop-ups-theyre-making-a-resurgence/index.md

  - css
  - html
date: 2022-09-13
last_updated: 2022-11-10
---

{% Aside 'caution' %}
The Open UI initiative resolved to change the attribute from `popup` to `popover` for the Popover API on [October 27, 2022](https://github.com/openui/open-ui/issues/627).
This [change request](https://chromium-review.googlesource.com/c/chromium/src/+/3991667) was merged into Chromium on [November 3, 2022](https://chromium-review.googlesource.com/c/chromium/src/+/3991667). References have been updated to make use of the new `popover` naming convention.
{% endAside %}

The goal of the [Open UI initiative](https://open-ui.org/) is to make it easier for developers to make great user experiences. To do this, we are trying to tackle the more problematic patterns that developers face. We can do this by providing better platform built-in APIs and components.

> One such problem area is pop-ups. Or as they are referred to in Open UI, "Popovers".

Author: @rachelandrew – Do you perhaps have a better way to word this? Basically, the Open UI has decided to rename the API and references towards it. But, people are naturally going to refer to this still as pop-ups for some part. I want to somehow sum that up in a line here.

> One such problem area is pop-ups. Or as they are referred to in Open UI, "Popovers".

Reviewer: ```suggestion
One such problem area is pop-ups, described in Open UI as "Popovers".
```
